packages/ui/src/components/va-navbar/VaNavbar.vue

```diff
@@ -4,19 +4,19 @@
     class="va-navbar"
     :style="computedStyle"
   >
-    <div class="va-navbar__content">
-      <div v-if="$slots.left" class="va-navbar__left">
-        <slot name="left" />
-      </div>
-
-      <div v-if="$slots.center" class="va-navbar__center">
-        <slot name="center" />
-      </div>
-
-      <div v-if="$slots.right" class="va-navbar__right">
-        <slot name="right" />
-      </div>
+    <div v-if="$slots.left" class="va-navbar__left">
+      <slot name="left" />
     </div>
+
+    <div v-if="$slots.center || $slots.default" class="va-navbar__center">
```

Comment:
Do we need both center and default to prevent breaking change? If yes - we should follow with deprecation.